OneCommunityGlobal/HighestGoodNetworkApp
# Mengtian_Notification_unittest

# Description
Unit Test for notificationCard.jsx


## Main changes explained:
- Add a unit test for notificationCard.jsx

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. do  `npm test notification.test.js `

## Screenshots or videos of changes:

<img width="701" alt="Screenshot 2024-06-13 at 5 39 04 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/122494725/f292c17f-e7c0-4500-a829-46a5395e56c6">
